src/UI/examples/Button/Standard/with_js_binding.php

```diff
@@ -0,0 +1,14 @@
+<?php
+function with_js_binding() {
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    return $renderer->render(
+		$f->button()->standard("Goto ILIAS", "#")
```

Comment:
The # navigates to the top of the screen in ILIAS, I think we should allow buttons not to have an action at all for those instances.